esp8266-fastled-webserver/include/simplehacks/static_eval.h

```diff
@@ -0,0 +1,50 @@
+/**
```

Comment:
SIMPLE FILE REVIEW

Adds ability to ensure compile-time evaluation of values.
Avoids variability caused by compilers that do not evaluate some expressions at compilation time.